components/InputFormGuest.js

```diff
@@ -11,11 +11,57 @@ export default function AddGuest() {
           <Input
             className="SmallFontStyle"
             type="text"
-            id="contactData"
-            name="contactData"
+            id="firstName"
+            name="firstName"
             required
+            placeholder="Vorname"
+          />
+          <Input
+            className="SmallFontStyle"
+            type="text"
+            id="lastName"
+            name="lastName"
+            required
+            placeholder="Nachname"
+          />
+          <Input
+            className="SmallFontStyle"
+            type="text"
+            id="address"
+            name="address"
+            required
+            placeholder="Anschrift"
+          />
+          <label className="MediumFontStyle" htmlFor="companion">
+            Begleitpersonen
+          </label>
+          <StyledDiv className="SmallFontStyle">Weiterer Gast</StyledDiv>
+          <StyledDiv className="SmallFontStyle">Kinder</StyledDiv>
```

Comment:
What's the purpose of these two `div`s? Can I add more guests or children somewhere?

Reply:
Yes, you can add guests and children. But I´m not done so there are not buttons yet.